source/_components/azuredns.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "Azure DNS"
+description: "Keep your Azure DNS IP address in sync with your external IP address"
+date: 2019-02-23 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: azure.png
+ha_category: Network
+featured: false
+ha_release: 0.90
+---
```

Comment:
What is the `ha_iot_class`?

Reply:
https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things

Reply:
Good one @klaasnicolaas. This component is not communicating with an IoT device, so I'm not sure if this needs to be specified. I've checked the [Namecheap component](https://github.com/home-assistant/home-assistant.io/blob/next/source/_components/namecheapdns.markdown) and the [DuckDNS component](https://github.com/home-assistant/home-assistant.io/blob/next/source/_components/duckdns.markdown) which offers similar functionalities, and I don't see the `ha_iot_class` specified there.

Can you confirm if this is required? Thanks.